src/async_client/client.py

```diff
@@ -91,7 +92,8 @@ async def wrapped(method, path, **kwargs):
     async def _request(self, method, path, **kwargs):
         return (await self._raw_request(method, path, **kwargs)).json(parse_float=Decimal)
 
-    async def _find_all(self, find_function, request, sort_field: str = 'id', items_field: str = 'items'):
+    async def _find_all(self, find_function, request, sort_field: str = 'id',
+                        items_field: str = 'items', batch_size: Optional[int] = None):
         result = await find_function(request, sort=[sort_field])
```

Comment:
```suggestion
        result = await find_function(request, sort=[sort_field], limit=batch_size)
```
`batch_size` should be passed to the "find_function" in the async client too